.shippable.yml

```diff
@@ -44,7 +44,8 @@ build:
           fi;
       - >
           if [ "$RUN_COMPLIANCE" = "1" -a "$IS_PULL_REQUEST" = "true" ]; then
-            export COMMIT_RANGE=origin/${PULL_REQUEST_BASE_BRANCH}..${COMMIT}
+            ARCH="-a x86";
```

Comment:
If we are effectively overwritting what the matrix would set ARCH= to, lets clean this up and make it more like based on something like SLAVE=1, SLAVE=2, etc.  (I'm happy for a better name than SLAVE)

So we'd end up with:

matrix:
  - SLAVE="1"
  - SLAVE="2"

....

if [ "$IS_PULL_REQUEST" = "true" ]; then
   if [ "SLAVE" = "1"]; then
      ARCH="-a x86";
   else
      ARCH="-a arm -a arc -a nios2 -a riscv32";
   fi;